lib/render.go

```diff
@@ -5,25 +5,47 @@ package gostatic
 
 import (
 	"bytes"
+	"fmt"
+	"strings"
+
+	chroma "github.com/alecthomas/chroma/formatters/html"
 	"github.com/yuin/goldmark"
-    "github.com/yuin/goldmark/extension"
-    "github.com/yuin/goldmark/parser"
-    "github.com/yuin/goldmark/renderer/html"
+	highlighting "github.com/yuin/goldmark-highlighting"
+	"github.com/yuin/goldmark/extension"
+	"github.com/yuin/goldmark/parser"
+	"github.com/yuin/goldmark/renderer/html"
 )
 
+func Markdown(source string, args []string) string {
+
+	extensions := []goldmark.Extender{
+		extension.Table,
+		extension.Strikethrough,
+		extension.Linkify,
+		extension.TaskList,
+		extension.GFM,
+		extension.DefinitionList,
+		extension.Footnote,
+		extension.Typographer,
+	}
+
+	for _, v := range args {
+		//chroma=monokai is a code highlighting style example
+		if strings.HasPrefix(v, "chroma=") {
+			style := strings.Replace(v, "chroma=", "", 1)
```

Comment:
If you have ideas how to improve this, let me know, otherwise, I'm not sure how or if I should change this.

I suppose I could have done `if len(args>=1) && strings.HasPrefix(args[0].....`, but I just kind of wrote it in a way where it can be expanded if more args get added later. 

Let me know if you think this should change. 😄

Reply:
I think this is good for now. :)